package.json

```diff
@@ -42,7 +45,12 @@
   },
   "private": true,
   "devDependencies": {
-    "autoprefixer": "^6.3.6"
+    "autoprefixer": "^6.3.6",
+    "babel-eslint": "^7.0.0",
+    "eslint": "^3.8.0",
+    "eslint-config-meteor": "0.0.9",
+    "eslint-plugin-babel": "^3.3.0",
+    "eslint-plugin-meteor": "^4.0.1"
   },
```

Comment:
As we are using React, do we need to add `eslint-plugin-react` and `eslint-plugin-jsx-a11y`? Or is already taken in care in the dependencies of the other plugins?